README.md

```diff
@@ -0,0 +1,69 @@
+# hello-kubernetes
+Example end-to-end build, test, and deploy pipelines for Kubernetes, using Google Cloud Build.
+
+# Instructions
+
+If you are new to Google Cloud Build, we recommend you start by visiting the [manage resources page](https://console.cloud.google.com/cloud-resource-manager) in the Cloud Console, [learn how to enable billing](https://cloud.google.com/billing/docs/how-to/modify-project), [enable the Cloud Build API](https://console.cloud.google.com/flows/enableapi?apiid=cloudbuild.googleapis.com), and [install the Cloud SDK](https://cloud.google.com/sdk/docs/).
```

Comment:
suggest linking to the quick start for Go: https://cloud.google.com/cloud-build/docs/quickstart-go

Reply:
Done